cli/command/formatter/service.go

```diff
@@ -41,10 +41,14 @@ Placement:
 {{- if .HasUpdateConfig }}
 UpdateConfig:
  Parallelism:	{{ .UpdateParallelism }}
-{{- if .HasUpdateDelay -}}
```

Comment:
The `-` tells the template engine to remove whitespace that follows. This was incorrect because it ended up merging Delay onto the same line as Parallelism. Notice that the other conditionals don't have a `-` at the end.

I could have opened another PR for this, but found it easiest to bundle it with the other changes I was making to the template.